### DISCUSSION TITLE
An option to set a font fallback when the custom font is not available

### DISCUSSION THREAD

**Discussion by spcbfr** [monkeytypegame/monkeytype#1605] [Ideas] [answered]
I have set my font to Operator Mono and I have that font installed so it works great, but sometimes I need to log in from another computer that does not have that font, so I end up updating my settings just for that one time where I need to use another person's PC, so I think it would be nice, if the user could set a fallback font that is one of the default fonts on monkeytype (say Fira Code) and it would use that if the custom font is not available.

**Comment by Miodec** [comment, marked as answer]
Added in 75774c9fb6e4accbd8f94ccfc4b7c70abc69721d
